app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = [Person(name=person["name"],
+                          age=person["age"])
+                   for person in people]
+    for person in people:
+        name = person["name"]
```

Comment:
You could retrieve here not only name, but rather instance
```suggestion
        person_instance = Person.people.get(person["name"])
```